script.js

```diff
@@ -87,7 +88,14 @@ restaurantsApp.getReviews=function(businessIDs){
 }
 
 restaurantsApp.init = function () {
-    restaurantsApp.getRestaurantIDs('Chicken', 'Toronto')
+    //set up event listener to accept user input
```

Comment:
Great. Its just this piece of code should be written in a separate function to be called in init